### DISCUSSION TITLE
1.0.1 - Rock and roll

### DISCUSSION THREAD

**Discussion by d8ahazard** [d8ahazard/sd_dreambooth_extension#979] [Announcements]
**Feb 28, 2023 Patch Notes**
- :seedling: Seeding | repeat training should produce exact^ model outputs (^similar, if using xformers and/or 8bit adam).

- 🦁  New optimizer | to use, set *Optimizer* to Lion on the **Settings** tab. recommended to **reduce LR to 1/10 of Adam's LRs** with Lion.

- :loud_sound: Offset noise | recommended value: **0.05-0.1**, may improve high/low lighting output pictures.

- :chart_with_upwards_trend: Improved graphs

- :stopwatch: DEIS Scheduler | new and improved schedule. To use for class pic generation uncheck *Generate Classification Images Using txt2img* on the **Settings** tab. To use as training scheduler, check *Use DEIS for noise scheduler* on the **Testing** tab

- :ok_woman: New Lora features | added Lora Dropout & Conv2d support and custom scaling of ranks for high quality. See cloneofsimo/lora#133 for more info.

- 🥼 Quality of Life improvements | Add version check notifications to the UI, along with changelogs.

- 🔦 Removed Flash Attention | In preparation for the release of torch 2, Flash attention has been removed as it's no longer supported by diffusers. When Torch2 is officially released, xformers will be removed as well, as it is supported natively in Torch2!

-  :bug: Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/d8ahazard/sd_dreambooth_extension/releases/tag/'>1.0.1 - Rock and roll</a>.</em>

**Comment by Arilziem** [comment]
Memory management should also work better for people with GPUs in the lower VRAM range, so if you weren't able to generate sample images recently, this might work now.
